### DISCUSSION TITLE
After canvas.render method I cannot move the assets and the widget toolbar disappears

### DISCUSSION THREAD

**Discussion by elvismontezumakw** [GrapesJS/grapesjs#4194] [Q&A]
Hello All.
I am new with GarpesJs.
I am facing an issue.
I have a web application in react in which the user drags and drops some blocks (widgets) to a canvas, then the user can move and reorder the widgets dragging and dropping them, also any widget shows a small toolbar with main actions as copy, remove, etc.. 

The widgets have some properties as images and texts but I did not find any other method to refresh the changes that the `editor.current.Canvas.render()` method, the problem with this method is that after apply it, I cannot reorder/move the widgets over the canvas, also I cannot see the toolbar anymore.

**Comment by artf** [comment]
Show a demo of your use case. I don't see why you should ever use `Canvas.render()`
